src/pages/scientificServices/pipelines/utils/input-utils.ts

```diff
@@ -7,15 +7,18 @@ export interface PipelineInputDescription {
   label?: string;
   placeholder?: string;
   helpText?: string;
+  validationRegex?: string;
 }
 
 export const INPUT_DESCRIPTIONS: Record<string, PipelineInputDescription> = {
   outputBasename: {
     label: 'Enter prefix for output file',
     placeholder: 'Enter prefix name',
     helpText: 'May only contain alphanumeric characters, dashes, and underscores.',
+    validationRegex: '^[a-zA-Z0-9_-]+$',
   },
   multiSampleVcf: {
     label: 'Select a multi-sample VCF file',
+    validationRegex: String.raw`^[^\s]+$`,
```

Comment:
is there a reason this regex is not the same as the outputBasename regex.  Also curious if we want to through the file suffix here too even though the file selector probably handles that for us already

Reply:
good call, i just made the regexes the same. 

i kind of like keeping the file suffix separate because it separates the type and name validation a little more clearly, but i honestly don't feel too strongly!